repose-aggregator/components/services/http-client-service/http-client-service-impl/src/main/java/org/openrepose/core/services/httpclient/impl/HttpConnectionPoolProvider.java

```diff
@@ -75,7 +75,7 @@ public static HttpClient genClient(String configRoot, PoolType poolConf) {
         params.setParameter(CoreConnectionPNames.MAX_HEADER_COUNT, poolConf.getHttpConnectionMaxHeaderCount());
         params.setParameter(CoreConnectionPNames.MAX_LINE_LENGTH, poolConf.getHttpConnectionMaxLineLength());
         params.setParameter(CoreConnectionPNames.SOCKET_BUFFER_SIZE, poolConf.getHttpSocketBufferSize());
-        params.setBooleanParameter(CHUNKED_ENCODING_PARAM, poolConf.isChunkedEncoding());
+        params.setParameter(CHUNKED_ENCODING_PARAM, poolConf.getChunkedEncoding());
```

Comment:
Seems like right here would be a good spot to filter out the 1 and 0, so your other code doesn't have to translate, but can just deal with values it understands.